app/services/UpscanService.scala

```diff
@@ -39,7 +39,7 @@ class UpscanService @Inject()(
 
     val success = controllers.routes.CsvFileUploadController.success(uploadId)
     val failure = controllers.routes.CsvFileUploadController.failure()
-    val upscanInitiateRequest = UpscanInitiateRequest(callback, urlToString(success), urlToString(failure))
+    val upscanInitiateRequest = UpscanInitiateRequest(callback, urlToString(success), urlToString(failure), 1, 100000000)
```

Comment:
won't setting max file size to 100000000 mean it's actually only allowing files up to 95mb large? I've just checked and aws seems to use binary bytes (i.e. 1024bytes to kb etc)